src/main/java/com/conveyal/gtfs/loader/JdbcTableWriter.java

```diff
@@ -1563,71 +1591,75 @@ private void updateReferencingTables(
+                                                    }
+                                                }
                                             }
+                                            String message = String.format(
+                                                "Cannot delete %s %s=%s. %d %s reference this %s.",
+                                                entityClass.getSimpleName(),
+                                                keyField.name,
+                                                keyValue,
+                                                result,
+                                                referencingTable.name,
+                                                entityClass.getSimpleName()
+                                            );
+                                            if (patternAndRouteIds.size() > 0) {
+                                                // Append referenced patterns data to the end of the error.
+                                                message = String.format(
+                                                    "%s\nReferenced patterns: [%s]",
+                                                    message,
+                                                    StringUtils.join(patternAndRouteIds, ",")
+                                                );
+                                            }
+                                            LOG.warn(message);
+                                            throw new SQLException(message);
                                         }
                                     }
-                                    String message = String.format(
-                                            "Cannot delete %s %s=%s. %d %s reference this %s.",
-                                            entityClass.getSimpleName(),
-                                            keyField.name,
-                                            keyValue,
-                                            result,
-                                            referencingTable.name,
-                                            entityClass.getSimpleName()
-                                    );
-                                    if (patternAndRouteIds.size() > 0) {
-                                        // Append referenced patterns data to the end of the error.
-                                        message = String.format(
-                                            "%s\nReferenced patterns: [%s]",
-                                                message,
-                                                StringUtils.join(patternAndRouteIds, ",")
-                                        );
-                                    }
-                                    LOG.warn(message);
-                                    throw new SQLException(message);
+                                    LOG.info("{} reference(s) in {} {}D!", result, refTableName, sqlMethod);
```

Comment:
Is the `D` intentional? 
```suggestion
                                    LOG.info("{} reference(s) in {} {}!", result, refTableName, sqlMethod);
```

Reply:
Yep! The SQL method is either UPDATE or DELETE and the 'D' makes it past tense.